src/test/java/io/jenkins/plugins/analysis/core/views/LocalizedPriorityTest.java

```diff
@@ -0,0 +1,146 @@
+    }
+
+    /**
+     * Verifies that the expected string is correct. This test case checks for negative results.
+     */
+    @Test
+    void shouldGetLocalizedStringNormalPriorityForNullNegativeTestCase() {
+        assertThat(LocalizedPriority.getLocalizedString(null)).isNotEqualTo(Messages.Priority_High());
+    }
+
+    /**
+     * Verifies that the expected string is correct. This test case checks for positive results.
+     */
+    @Test
+    void shouldGetLongLocalizedStringHighPriorityPositiveTestCase() {
+        assertThat(LocalizedPriority.getLongLocalizedString(Priority.HIGH)).isEqualTo(Messages.HighPriority());
+    }
+
+    /**
+     * Verifies that the expected string is correct. This test case checks for negative results.
+     */
+    @Test
+    void shouldGetLongLocalizedStringHighPriorityNegativeTestCase() {
+        assertThat(LocalizedPriority.getLongLocalizedString(Priority.HIGH)).isNotEqualTo(Messages.LowPriority());
+    }
+
+    /**
+     * Verifies that the expected string is correct. This test case checks for positive results.
+     */
+    @Test
+    void shouldGetLongLocalizedStringNormalPriorityPositiveTestCase() {
+        assertThat(LocalizedPriority.getLongLocalizedString(Priority.NORMAL)).isEqualTo(Messages.NormalPriority());
+    }
+
+    /**
+     * Verifies that the expected string is correct. This test case checks for negative results.
+     */
+    @Test
+    void shouldGetLongLocalizedStringNormalPriorityNegativeTestCase() {
+        assertThat(LocalizedPriority.getLongLocalizedString(Priority.NORMAL)).isNotEqualTo(Messages.LowPriority());
+    }
+
+    /**
+     * Verifies that the expected string is correct. This test case checks for positive results.
+     */
+    @Test
+    void shouldGetLongLocalizedStringLowPriorityPositiveTestCase() {
+        assertThat(LocalizedPriority.getLongLocalizedString(Priority.LOW)).isEqualTo(Messages.LowPriority());
+    }
+
+    /**
+     * Verifies that the expected string is correct. This test case checks for negative results.
+     */
+    @Test
+    void shouldGetLongLocalizedStringLowPriorityNegativeTestCase() {
+        assertThat(LocalizedPriority.getLongLocalizedString(Priority.LOW)).isNotEqualTo(Messages.NormalPriority());
+    }
+
+    /**
+     * Verifies that the expected string is correct. This test case checks for positive results.
+     */
+    @Test
+    void shouldGetLongLocalizedStringNormalPriorityForNullPositiveTestCase() {
+        assertThat(LocalizedPriority.getLongLocalizedString(null)).isEqualTo(Messages.NormalPriority());
+    }
+
+    /**
+     * Verifies that the expected string is correct. This test case checks for negative results.
+     */
+    @Test
+    void shouldGetLongLocalizedStringNormalPriorityForNullNegativeTestCase() {
+        assertThat(LocalizedPriority.getLongLocalizedString(null)).isNotEqualTo(Messages.HighPriority());
+    }
+
+}
```

Comment:
Die Klasse ist LocalizedPriority ist sehr einfach, da darf der Test auch einfach sein: Alles in eine (oder 2) Testmethoden packen und die isNotEqual raus.